redux/action.js

```diff
@@ -34,11 +37,41 @@ export const addShopListItem = (item) => {
   };
 };
 
-export const delShopListItem = (item) => {
+export const delShopListItem = (item, saveForLater = false) => {
   return {
     type: REMOVE_SHOPLIST_ITEMS,
     payload: {
       item,
+      saveForLater,
+    },
+  };
+};
+
+export const saveForLater = (item) => {
+  console.log({ item });
+  return {
+    type: SAVE_FOR_LATER,
+    payload: {
+      item,
+    },
+  };
+};
+
+// export const moveToCart = (item) => {
+//   return {
+//     type : MOVE_TO_CART,
+//     payload: {
+//       item
+//     }
+//   }
+// }
+
```

Comment:
Remove these comments if not needed

Reply:
They are needed for other feature that I have not pushed